src/pool-bin/libraries/BinPool.sol

```diff
@@ -317,15 +317,21 @@ library BinPool {
             bytes32 binReserves = self.reserveOfBin[id];
             uint256 supply = self.shareOfBin[id];
 
-            _subShare(self, params.from, id, params.salt, amountToBurn);
+            amountToBurn = _subShare(self, params.from, id, params.salt, amountToBurn);
```

Comment:
/// if user is last lp, amountToBurn would be amountToBurn + MINIMUM_SHARE 
amountToBurn = _subShare(self, params.from, id, params.salt, amountToBurn);